src/assignment/assignment.service.ts

```diff
@@ -96,7 +96,9 @@ export class AssignmentService {
 								assignment_label: { label: { _ilike: `%${trimmedFilterString}%` } },
 							},
 						},
-						{ assignment_type: { _ilike: `%${trimmedFilterString}%` } },
+						...(!canEditAssignments // Only search by teacher if user is not a teacher
+							? [{ owner: { full_name: { _ilike: `%${trimmedFilterString}%` } } }]
```

Comment:
`owner` => `author`?

Reply:
in de database zit de author name onder owner.full_name
de column id is 'author'

Ik heb het hernoemd om meer in lijn te liggen met de andere tabellen (collectie, content pages, ...)